salt/client/api.py

```diff
@@ -120,7 +120,7 @@ def run(self, cmd):
         '''
         cmd = dict(cmd)  # make copy
         client = 'minion'  # default to local minion client
-        mode = cmd.get('mode', 'async')  # default to 'async'
```

Comment:
I don't think we need to change this line here, and won't this break the api?

Reply:
Yeah, you're right. I looked on it again, and this formats `something_async`, which we never changed to `something_asynchronous`. Fixed!